cilantro_audit/admin_page.py

```diff
@@ -0,0 +1,10 @@
+from kivy.uix.screenmanager import Screen
+
+from cilantro_audit.constants import COMPLETED_AUDITS_LIST_PAGE
+
+
+class AdminPage(Screen):
+    pass
+
+    def load_completed_audits_list_page(self):
```

Comment:
Sorry if I wasn't being clear! I think this entire function can be deleted. In the `admin_page.kv` file, it seems that the transition to view the completed audits is already set via `root.manager.current = COMPLETED_AUDITS_LIST_PAGE`.

Reply:
Ah yes. You're right. Thank you.